packages/hash/api/src/collab/Instance.ts

```diff
@@ -81,6 +77,7 @@ export class Instance {
   lastActive = Date.now();
   waiting: Waiting[] = [];
   saveChain = Promise.resolve();
+  collecting: ReturnType<typeof setTimeout> | null = null;
```

Comment:
@nathggns there was a git conflict here, which I resolved by adding lines from both versions. Now this looks a bit wrong to me, I should have probably deleted `collecting` 😅

Reply:
UPD: Attempt #2 — hope it is fine now. Sorry for introducing the mess!